roles/marathon/meta/main.yml

```diff
@@ -122,6 +122,8 @@ galaxy_info:
   #- web
 dependencies:
   - role: consul
+  - role: registrator
```

Comment:
need to double check, but i think registrator might publish services that already exist into the consul backend anyway (when it starts up).